partitioned-table.md

```diff
@@ -1431,4 +1495,73 @@ mysql> explain select /*+ TIDB_INLJ(t1, t2) */ t1.* from t1, t2 where t2.code =
 
 From example 2, you can see that in `dynamic` mode, the execution plan with IndexJoin is selected when you execute the query.
 
-Currently, neither static nor dynamic pruning mode supports prepared statements plan cache. 
+Currently, neither `static` nor `dynamic` pruning mode supports prepared statements plan cache.
+
+#### Update statistics of partitioned tables in dynamic pruning mode
+
+1. Locate all partitioned tables:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select distinct concat(TABLE_SCHEMA,'.',TABLE_NAME)
+        from information_schema.PARTITIONS
+        where TABLE_SCHEMA not in('INFORMATION_SCHEMA','mysql','sys','PERFORMANCE_SCHEMA','METRICS_SCHEMA');
+    ```
+
+    ```
+    +-------------------------------------+
+    | concat(TABLE_SCHEMA,'.',TABLE_NAME) |
+    +-------------------------------------+
+    | test.t                              |
+    +-------------------------------------+
+    1 row in set (0.02 sec)
+    ```
+
+2. Generate the statements for updating the statistics of all partitioned tables:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select distinct concat('ANALYZE TABLE ',TABLE_SCHEMA,'.',TABLE_NAME,' ALL COLUMNS;')
+        from information_schema.PARTITIONS
+        where TABLE_SCHEMA not in ('INFORMATION_SCHEMA','mysql','sys','PERFORMANCE_SCHEMA','METRICS_SCHEMA');
+    +----------------------------------------------------------------------+
+    | concat('ANALYZE TABLE ',TABLE_SCHEMA,'.',TABLE_NAME,' ALL COLUMNS;') |
+    +----------------------------------------------------------------------+
+    | ANALYZE TABLE test.t ALL COLUMNS;                                    |
+    +----------------------------------------------------------------------+
+    1 row in set (0.01 sec)
+    ```
+
+    You can change `ALL COLUMNS` to the columns you need.
+
+3. Export the batch update statements to a file:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    mysql --host xxxx --port xxxx -u root -p -e "select distinct concat('ANALYZE TABLE ',TABLE_SCHEMA,'.',TABLE_NAME,' ALL COLUMNS;') \
+        from information_schema.PARTITIONS \
+        where TABLE_SCHEMA not in ('INFORMATION_SCHEMA','mysql','sys','PERFORMANCE_SCHEMA','METRICS_SCHEMA');" | tee gatherGlobalStats.sql
+    ```
+
+4. Execute a batch update:
+
+    Process SQL statements before executing the `source` statement:
```

Comment:
```suggestion
    Process SQL statements before executing the `source` command:
```

Reply:
`source` is a MySQL client command.
https://dev.mysql.com/doc/refman/8.0/en/mysql-commands.html